public/server/db.json

```diff
@@ -11,5 +11,58 @@
 
     "idk":{
         "definition": "I Don't Know"
+    },
+
+    "imo":{
+        "definition": "I My Opinion"
```

Comment:
@slim-codes I think there is a typo here. It should be `In my opinion`

Reply:
@Olanetsoft Nice catch. Thanks for that. I've fixed it